clabe.js

```diff
@@ -2,661 +2,3724 @@
 
 const clabe = {
 
-   version: '[VERSION]',
+  version: '[VERSION]',
```

Comment:
The indentation for this project is 3 spaces.

Reply:
Ok, I'm going to change the identation to 3 spaces